### PR TITLE
Jenkins File and OpenShift template

### PR DESCRIPTION
**Pull request by harjinder-hari**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+if (env.BRANCH_NAME == 'master') {
+    node('oc') {
+
+        def dcs = ['license-analysis' ]
+        lock('f8a_staging') {
+
+            stage('Deploy - stage') {
+                unstash 'template'
+                sh "oc --context=rh-idev process -v IMAGE_TAG=${commitId} -v CPU_REQUEST=2 -f template.yaml | oc --context=rh-idev apply -f -"
```
Comment (miteshvp) [inline Jenkinsfile]: do we need 2vCPUs?

File: openshift/template.yaml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      targetPort: 0
+    selector:
+      service: license-analysis
+
+parameters:
+- description: CPU request
+  displayName: CPU request
+  required: true
+  name: CPU_REQUEST
+  value: "3.5"
```
Comment (miteshvp) [inline openshift/template.yaml]: we need to re-visit this value for CPU_REQUEST and CPU_LIMIT . I guess they are too high for license scoring.

File: Jenkinsfile (hunk 1 above)
Comment (harjinder-hari) [inline Jenkinsfile]: Done. Reduced it to 1.

File: openshift/template.yaml (hunk 2 above)
Comment (harjinder-hari) [inline openshift/template.yaml]: Done. Reduced them to 1.

**Comment by harjinder-hari** [comment]
Hi @miteshvp , I have addressed your review requests. Can you pls merge the PR ?

**Pull request merged**